packages/server/src/tests/header/headerSelection.ts

```diff
@@ -107,91 +91,65 @@ const supportersTest: HeaderTest = {
     ],
 };
 
-const supportersTestUS: HeaderTest = {
-    name: 'header-supporter__USEOY',
-    audience: 'AllExistingSupporters',
-    variants: [
-        {
-            name: 'control',
-            modulePathBuilder: usEOYHeader.endpointPathBuilder,
-            content: {
-                heading: 'Thank you for supporting us',
-                subheading: '',
-                primaryCta: {
-                    url:
-                        'https://support.theguardian.com/us/contribute?selected-contribution-type=ONE_OFF',
-                    text: 'Make an extra contribution',
-                },
-            },
-            mobileContent: {
-                heading: 'Thank you',
-                subheading: '',
-                primaryCta: {
-                    url:
-                        'https://support.theguardian.com/us/contribute?selected-contribution-type=ONE_OFF',
-                    text: 'Contribute again',
-                },
-            },
-        },
-    ],
-};
+const isNotReportedAsSupporter = ['AllNonSupporters'];
+
+// Again, exported for Jest testing
+export const selectBestTest = (
+    targeting: HeaderTargeting,
+    allTests: HeaderTest[],
+): HeaderTestSelection | null => {
+    const { showSupportMessaging, countryCode } = targeting;
 
-const usEoyPeriodStart = new Date(2021, 10, 22);
-const usEoyPeriodEnd = new Date(2022, 1, 1);
+    const countryGroupId = countryCodeToCountryGroupId(countryCode.toUpperCase());
 
-const isInUsEoyPeriod = (date: Date): boolean => {
-    return isAfter(date, usEoyPeriodStart) && isBefore(date, usEoyPeriodEnd);
-};
+    const selectedTest: HeaderTest | undefined = allTests.find(test => {
+        const { isOn, userCohort, locations } = test;
+
+        if (!isOn) {
+            return false;
+        }
+
+        if (showSupportMessaging && !isNotReportedAsSupporter.includes(userCohort)) {
+            return false;
+        }
 
-const hasNotContributedDuringUsEoyPeriod = (lastOneOffContributionDate?: string): boolean => {
-    if (!lastOneOffContributionDate) {
+        if (!showSupportMessaging && isNotReportedAsSupporter.includes(userCohort)) {
+            return false;
+        }
+
+        if (!locations.includes(countryGroupId)) {
```

Comment:
prefer existing [inCountryGroups function](https://github.com/guardian/support-dotcom-components/blob/bbc5ea13de393df26ca76fdb3a3b9ee33114d0a4/packages/shared/src/lib/geolocation.ts#L386), which handles an empty array as global

Reply:
Recoded to use suggested function